### DISCUSSION TITLE
Add support for Panorama / 360 photos

### DISCUSSION THREAD

**Discussion by vishnukvmd** [ente-io/ente#828] [Enhancements]

**Comment by MichalSvatos** [comment]
Any updates regarding this issue? I have recently become a paying customer and truly miss this feature. Thank you.

Edit: Also duplicate - https://github.com/ente-io/ente/discussions/522

**Comment by dumbbell** [comment]
Likewise. I also became a paying customer, but I still have my 360 photos over at Google for work because this is the only way I can share them with clients.

**Comment by hb0nd** [comment]
these are supported already at least on android

**Comment by dumbbell** [comment]
Indeed, I just tested and it worked with the Android application. I’m looking forward to have the same feature in a browser!
